### DISCUSSION TITLE
javascript runtime

### DISCUSSION THREAD

**Discussion by wenq1** [vlang/v#18664] [Ideas]
### Describe the feature

C++ has nodejs, Rust had Deno, zig has bun.js. What do v plan to have?

### Use Case

JavaScript runtime

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### Version used

Latest

### Environment details (OS name and version, etc.)

All

**Comment by JalonSolov** [comment]
Looks at the output of `v help build-js`.  That's what is implemented so far.
